src/main/java/database/service/service/JsonDatabaseService.java

```diff
@@ -0,0 +1,272 @@
+package database.service.service;
+
+import code.practice.exceptions.database.CreationDatabaseException;
+import code.practice.exceptions.database.DatabaseDoesNotExistException;
+import code.practice.exceptions.database.DeletionDatabaseException;
+import code.practice.exceptions.database.DeserializeDatabaseException;
+import code.practice.exceptions.database.IdDoesNotExistException;
+import code.practice.exceptions.database.SerializeDatabaseException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import database.service.entity.BaseEntity;
+import database.service.helper.Settings;
+import database.service.helper.Validator;
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class JsonDatabaseService implements DatabaseService {
+    private final Settings settings;
+    private final ObjectMapper objectMapper;
+    static final String EMPTY_BRACKETS_TO_JSON = "[]";
+    private Map<String, Integer> entityIds;
+
+    public JsonDatabaseService() throws CreationDatabaseException {
+        settings = new Settings();
```

Comment:
Consider adding another constructor where we expect database propery file name for `Settings`.